### ISSUE TITLE
Mod: # added passive skills are jewel sockets (cluster jewel)

### ISSUE THREAD

**Issue by ZeldaZocker** [PoE-Overlay-Community/PoE-Overlay-Community-Fork#111]
**Describe the bug**
The "# added passive skills are jewel sockets" mod on cluster jewels isn't listed in the overlay and can't be searched.

1. Price-check a cluster jewel with the "# added passive skills are jewel sockets" mod

**Expected behavior**
The mod is listed and searchable for

**Screenshots**
![PoE](https://user-images.githubusercontent.com/34456207/86759700-fd684680-c044-11ea-8b3f-776927ff030a.png)

## 🌍  Your Environment

- OS: Win10 x64
- Version:  e.g. 0.7.1 installed
- PoE:  e.g. Steam 3.11.1 English

**Comment by WhiteFang5** [comment]
Can you copy-paste the item in a reply please? (in a code snipped, so formatting is preserved)

**Comment by ZeldaZocker** [comment]
I don't have the jewel from the screenshot anymore but here is another one with the same problem:
```
Rarity: Rare
Dragon Essence
Large Cluster Jewel
--------
Item Level: 61
--------
Adds 8 Passive Skills (enchant)
2 Added Passive Skills are Jewel Sockets (enchant)
Added Small Passive Skills grant: 12% increased Lightning Damage (enchant)
--------
Added Small Passive Skills also grant: 2% increased Damage
Added Small Passive Skills also grant: +3% to Lightning Resistance
1 Added Passive Skill is Storm Drinker
--------
Place into an allocated Large Jewel Socket on the Passive Skill Tree. Added passives do not interact with jewel radiuses. Right click to remove from the Socket.
```

**Comment by WhiteFang5** [comment]
The issue should be resolved in the next version:
![image](https://user-images.githubusercontent.com/4527188/87139850-dd34c380-c2a0-11ea-94b3-606e5eab1d8d.png)

**Comment by WhiteFang5** [comment]
Fixed in v0.7.3
